integrationTests/testInitializer.go

```diff
@@ -1321,3 +1324,89 @@ func StartP2pBootstrapOnProcessorNodes(nodes []*TestProcessorNode) {
 	fmt.Println("Delaying for nodes p2p bootstrap...")
 	time.Sleep(p2pBootstrapStepDelay)
 }
+
+func MakeDummyGasSchedule() map[string]uint64 {
+	gasSchedule := make(map[string]uint64)
+
+	gasSchedule = arwenconfig.FillGasMapWithWASMOpcodeValues(gasSchedule, 1)
+	gasSchedule["StorePerByte"] = 1
```

Comment:
make a function for these API calls as well in arwen.

Reply:
Done.